codestodians/komlie-frontend
# [Feature] Make Cart Page Responsive

#### What does this PR do?
- Makes the cart page responsive on all viewports

#### How should this be manually tested?
- Check out into the `feature/make-cart-page-responsive-167378368` branch
- Navigate to the cart page
- Check for responsiveness on all viewports

#### What are the relevant pivotal tracker stories?
- [#167378368](https://www.pivotaltracker.com/story/show/167378368)
